templates/dataset_dag/sambegin.sh

```diff
@@ -50,16 +91,17 @@ chmod +x ${JSB_TMP}/ifdh.sh
 ################################################################################
 ## main ()                                                            ##########
 ################################################################################
-echo `date` BEGIN executing /fife/local/scratch/uploads/fermilab/mengel/2020-01-14_162931.318256_6730/fife_wrap_20200114_162957_3316054.sambegin.sh
->&2 echo `date` BEGIN executing /fife/local/scratch/uploads/fermilab/mengel/2020-01-14_162931.318256_6730/fife_wrap_20200114_162957_3316054.sambegin.sh
 #EXPERIMENT=$1
 #DEFN=$2
 #PRJ_NAME=$3
 #GRID_USER=$4
 
 export JSB_TMP=$_CONDOR_SCRATCH_DIR/jsb_tmp
 mkdir -p $JSB_TMP
+redirect_output_start
 setup_ifdh_env
+echo `date` BEGIN executing /fife/local/scratch/uploads/fermilab/mengel/2020-01-14_162931.318256_6730/fife_wrap_20200114_162957_3316054.sambegin.sh
```

Comment:
This isn't just for you, Kevin, but also for @marcmengel .  Why are we saying here that we're starting a script that's in Marc's area...on /fife/local?  This seems very wrong...and possibly a holdover from testing.  Please correct me if I'm wrong.

Reply:
Well that's embarrassing that I moved those lines but didn't notice that, good catch. Probably a "generated" script got mixed up with the template at some point. We could probably just change it to "BEGIN executing sambegin.sh"?

Reply:
That sounds good.

Reply:
changed